todo-app/src/App.js

```diff
@@ -1,13 +1,24 @@
-import React, { Component } from 'react';
+import React from 'react';
+import TodoList from './TodoList';
+import Todos from './Todos';
 
-class App extends Component {
+export default class App extends React.Component {
+  renderTodos(Todos) {
+    return Todos.map((todo, index) => {
+      return (
+        <TodoList
```

Comment:
A better naming for this component as it accepts a single `todo` object is `Todo`? What do you think?